integration_tests/mockApis/licence.ts

```diff
@@ -22,7 +22,7 @@ const licencePlaceholder = {
   sentenceStartDate: '10/01/2019',
   sentenceEndDate: '26/04/2022',
   licenceStartDate: '01/04/2021',
-  licenceExpiryDate: '26/04/2022',
+  licenceExpiryDate: '26/04/2060',
```

Comment:
Increased this to far in the future so that integration tests dont suddenly break on validation of the induction time